src/util/chroma/location.rs

```diff
@@ -15,6 +19,55 @@ pub enum Location {
     Bottom,
 }
 
+impl Location {
+    /// Returns the name of this location. Should usually return Some(name).
```

Comment:
In what situations would this return `None`?

Reply:
Well, the docs say that `av_chroma_location`
>     (returns) the name for provided chroma location or NULL if unknown. 

Looking at the [implementation](https://ffmpeg.org/doxygen/trunk/pixdesc_8c_source.html#l03542) though, unless you pass AVCHROMA_LOC_NB (not part of ABI) or an integer (I think this is valid in C? but UB in Rust), this always returns a non-null ptr.

But since the docs don't guarantee it and this could change with any update, I went with the Option<T> to be safe.